requirements/lite/base.txt

```diff
@@ -2,7 +2,7 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 numpy>=1.17.2, <1.23.1
-torch>=1.9.*, <1.13.0
+torch>=1.9, <1.13.0
```

Comment:
Opening this comment to block the merge until there's confirmation for https://github.com/Lightning-AI/lightning/pull/15184#pullrequestreview-1146920075